src/bench/bench.cpp

```diff
@@ -88,7 +88,7 @@ bool benchmark::State::KeepRunning()
     lastCycles = nowCycles;
     ++count;
 
-    if (now - beginTime < maxElapsed) return true; // Keep going
+    if (now - beginTime < maxElapsed || count == 1) return true; // Keep going
```

Comment:
It might be better to fix the issue in the output at the bottom of the function like so:

```
     // Output results
-    double average = (now-beginTime)/count;
-    int64_t averageCycles = (nowCycles-beginCycles)/count;
+    double average = (count > 0) ? (now-beginTime)/count : 0.0;
+    int64_t averageCycles = (count > 0) ? (nowCycles-beginCycles)/count : 0;
     std::cout << std::fixed << std::setprecision(15) << name << "," << count << "," << minTime << "," << maxTime << "," << average << ","
               << minCycles << "," << maxCycles << "," << averageCycles << "\n";
```

I would say that this reads ok in isolation as just dumb output code and avoids having to understand the branching in function body above.